content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,836 @@
+         "min": 0.0,
+         "max": 1.0,
+         "connection": {
+         "type": "ShaderInput",
+         "name": "m_detailBranchAmplitude"
+         }
+      },
+      {
+         "name": "WindX",
+         "displayName": "Wind direction x",
+         "description": "Wind in the x direction. This would typically come from a wind system instead of a material property, but is here as a proof of concept.",
+         "type": "Float",
+         "defaultValue": 0.0,
+         "min": -1.0,
+         "max": 1.0,
+         "connection": {
+         "type": "ShaderInput",
+         "name": "m_windX"
+         }
+      },
+      {
+         "name": "WindY",
+         "displayName": "Wind direction y",
+         "description": "Wind in the y direction. This would typically come from a wind system instead of a material property, but is here as a proof of concept.",
+         "type": "Float",
+         "defaultValue": 0.0,
+         "min": -1.0,
+         "max": 1.0,
+         "connection": {
+         "type": "ShaderInput",
+         "name": "m_windY"
+         }
+      },
+      {
+         "name": "WindBendingStrength",
+         "displayName": "Bending strength",
+         "description": "Bending strength. This would typically come from a wind system instead of a material property, but is here as a proof of concept.",
+         "type": "Float",
+         "defaultValue": 0.0,
+         "min": 0.0,
+         "max": 7.0,
+         "connection": {
+         "type": "ShaderInput",
+         "name": "m_bendingStrength"
+         }
+      },
+      {
+         "name": "WindBendingFrequency",
+         "displayName": "Wind Bending Frequency",
+         "description": "The frequency that the object sways back and forth caused by the wind. This would typically come from a wind system instead of a material property, but is here as a proof of concept.",
+         "type": "Float",
+         "defaultValue": 0.0,
+         "min": 0.0,
+         "max": 1.5,
+         "connection": {
+         "type": "ShaderInput",
+         "name": "m_windBendingFrequency"
+         }
+      }
+   ]
+   }
+
+   ```
+
+1. Open `VegetationBending_Common.azsli`.
+
+1. Delete the previous offset variables if you haven't already, and declare the bending property variables in `MaterialSrg`:
+   
+   ```hlsl
+    float m_detailFrequency;
+    float m_detailLeafAmplitude;
+    float m_detailBranchAmplitude;
+    float m_windX;
+    float m_windY;
+    float m_bendingStrength;
+    float m_windBendingFrequency;
+    ```
+
+1. Open the **Editor**, then open the **Material Editor**, and choose to edit the leaf material (`aspen_leaf.material`). Make sure you select the leaf material because that is the parent material of the other parts of the tree.
+
+1. On the right, scroll down to **Vegetation Bending** and ensure that the 6 properties you just added are there.
+
+1. Adjust the properties! Make sure you adjust all of them so that you can see bending in the later steps.
```

Comment:
Are there any recommendations on what to adjust the properties to?

Reply:
Changed this step to the following:

> 1. Adjust the properties! You can adjust them to the following to ensure you can see bending later:
>    * **Detail bending frequency** - `0.3`
>    * **Detail bending leaf amplitude** - `0.3`
>    * **Detail bending branch amplitude** - `0.3`
>    * **Wind direction x** - `0.5`
>    * **Wind direction y** - `0.5`
>    * **Bending strength** - `4.0`
>    * **Wind bending frequency** - `0.7`